### PR TITLE
Remove workarounds for dangling helix-fetch sockets

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

**Comment by stefan-guggisberg** [comment]
This PR is meant to verify that disabling `keep-alive` is not required anymore as of `helix-fetch` 1.4.1.

Hmm, seems like I was wrong :/. The post-deploy test is still hanging. I'll keep this PR open and will investigate further.

**Comment by stefan-guggisberg** [comment]
Weird, the tests seem to hang only sporadically. Now they succeeded.

**Comment by MarquiseRosier** [comment]
Oh wow now that I get to take a peek; i'm realizing the epsagon test didn't even have a keep alive: false flag set; I can submit a PR with it; for now. Supposing it works.

**Comment by stefan-guggisberg** [comment]
Yep, the epsagon test doesn’t use fetch.

File: src/index.js [hunk 1]
```diff
@@ -67,11 +67,11 @@ async function serviceembed(params, url, log) {
   });
 
   return fetch(api.href)
-    .then((data) => {
-      if (!data.ok) {
-        throw new Error(`Status ${data.status}: ${data.statusText || 'request failed, check your url'}`);
+    .then(async (resp) => {
```
Comment (tripodsan) [inline src/index.js]: this shouldn't make a difference, since the return value is already a promise

**Comment by stefan-guggisberg** [comment]
found the problem: https://github.com/adobe/helix-embed/blob/3bb08c451436ba9a7e5e3b8aafe2020840b1e4e0/src/index.js#L21-L22

**Pull request closed without merging**